contracts/src/security/pausing/interfaces/IPauseManager.sol

```diff
@@ -85,11 +91,21 @@ interface IPauseManager {
    */
   error IsPaused(PauseType pauseType);
 
+  /**
+   * @dev Thrown when unpauseDueToExpiry is attempted before a pause has expired.
```

Comment:
do we care if it is the security council unpausing? I don't think we do, just noting the edge case.

Reply:
no :) posting for public record - we don't want to add extra logic to `unPauseDueToExpiry` for the SECURITY_COUNCIL_ROLE case. And the SECURITY_COUNCIL_ROLE has the avenue to unpause via `unPauseByType`